draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -40,14 +39,24 @@ type VariantType =
 export type GuidanceBlockProps = {
   layout?: LayoutType
   illustration: React.ReactElement<SpotProps | SceneProps>
+  /*
+   ** Sets the how the width and aspect ratio will respond to the Illustration passed in.
+   */
   illustrationType?: IllustrationType
   text: {
     title: string
     description: string | React.ReactNode
   }
   smallScreenTextAlignment?: TextAlignment
   actions?: GuidanceBlockActions
+  /*
+   ** This will still require the secondary object to be passed into the actions ie: {secondary: { label: "Dismiss action" }}`
+   */
   secondaryDismiss?: boolean
+  /*
+   ** persistent should always return true and will soon be deprecated.
+   ** The X close icon has been superseded with the pattern "dismiss" or "cancel" using the secondary action.
+   */
```

Comment:
```suggestion
  /**
   * persistent should always return true and will soon be deprecated.
   * The X close icon has been superseded with the pattern "dismiss" or "cancel" using the secondary action.
   */
```

I think doc blocks are meant to look like this?
I'd probably also add backticks to `persistent` to indicate it's the prop name :)